### PR TITLE
Correcting misspelled words

### PR DESCRIPTION
**Pull request by moritamori**

### CODE REVIEW

File: custom_completions.go [hunk 1]
```diff
@@ -302,7 +302,7 @@ func (c *Command) getCompletions(args []string) (*Command, []string, ShellCompDi
 		if len(finalArgs) == 0 && !foundLocalNonPersistentFlag {
 			// We only complete sub-commands if:
 			// - there are no arguments on the command-line and
-			// - there are no local, non-peristent flag on the command-line or TraverseChildren is true
+			// - there are no local, non-persistent flag on the command-line or TraverseChildren is true
```
Comment (marckhouzam) [inline custom_completions.go:305]: Might as well put an 's' at 'flag' since we're correcting this sentence

**Review by marckhouzam — APPROVED** [review]
Thanks for this.

**Review by jpmcb — CHANGES_REQUESTED** [review]
Hi @moritamori - can you rebase and get the latest changes I just merged from #1192 (which include a file name change)

Also, if you could include the changes suggested in #1350, that would be highly appreciated!

**Comment by moritamori** [comment]
@jpmcb
Thank you for your feedback. 
I rebased this branch and inclueded the changes suggested in #1350.

**Comment by moritamori** [comment]
@jpmcb 
Could you accept?

**Review by jpmcb — APPROVED** [review]
LGTM - thanks for the patience on this!! :rocket:

**Pull request merged**